docs/ru/engines/table-engines/integrations/s3.md

```diff
@@ -11,21 +11,20 @@ toc_title: S3
 
 ``` sql
 CREATE TABLE s3_engine_table (name String, value UInt32) 
-ENGINE = S3(path, [aws_access_key_id, aws_secret_access_key,] format, structure, [compression])
+ENGINE = S3(path, [aws_access_key_id, aws_secret_access_key,] format, [compression])
 ```
 
 **Параметры движка**
 
 -   `path` — URL-адрес бакета с указанием пути к файлу. Поддерживает следующие подстановочные знаки в режиме "только чтение": `*`, `?`, `{abc,def}` и `{N..M}` где `N`, `M` — числа, `'abc'`, `'def'` — строки. Подробнее смотри [ниже](#wildcards-in-path).
 -   `format` — [формат](../../../interfaces/formats.md#formats) файла.
--   `structure` — структура таблицы в формате `'column1_name column1_type, column2_name column2_type, ...'`.
 -   `compression` — тип сжатия. Возможные значения: `none`, `gzip/gz`, `brotli/br`, `xz/LZMA`, `zstd/zst`. Необязательный параметр. Если не указано, то тип сжатия определяется автоматически по расширению файла. 
```

Comment:
May be also add `aws_access_key_id` and `aws_secret_access_key` to the list of parameters?

Reply:
It would be nice to describe it, but I don't know who to contact on this issue.

Reply:
Ok. i think you can add something like this:
> Access keys - long-term credentials for the AWS account user. They consist of two parts: an `access key ID` and a `secret access key`. Like a user name and password, you must use both the `access key ID` and `secret access key` together to authenticate your requests. They can are added as optional parameters of `s3 table engine`:
- `aws_access_key_id`
- `aws_secret_access_key`

Reply:
Thank you! I updated this.